src/java/com/unifina/signalpath/ModuleWithUI.java

```diff
@@ -72,17 +101,30 @@ private StreamService getStreamService() {
 		return streamService;
 	}
 
-	public void pushToUiChannel(Map content) {
-		Stream stream = getUiChannel().getStream();
+	private StreamMessage getMsg(Stream stream, Map content){
 		int streamPartition = StreamPartitioner.partition(stream, null);
+		long timestamp = System.currentTimeMillis();
```

Comment:
Never use system time in modules, can't work in historical playbacks. Always use event time available in `getGlobals().time`